### PR TITLE
--setup-create-override-cfg cli arg

### PR DESCRIPTION
**Pull request by KANAjetzt**
Adds cmd arg:

###  --setup-create-override-cfg
Forces the setup to create the ``override.cfg`` in the game base directory.
Skips the ``project.binary`` injection.

   
   
The project binary injection is skipped in that case.
```python
	if is_setup_create_override_cfg:
		handle_override_cfg()
	else:
		handle_project_binary()
```

### CODE REVIEW

**Review by otDan — APPROVED** [review]

File: addons/mod_loader/mod_loader_setup.gd [hunk 1]
```diff
@@ -58,7 +59,11 @@ func try_setup_modloader() -> void:
 	if is_loader_set_up() and not is_loader_setup_applied():
 		modloaderutils.log_info("ModLoader is set up, but the game needs to be restarted", LOG_NAME)
 		ProjectSettings.set_setting(settings.IS_LOADER_SETUP_APPLIED, true)
-		var _savecustom_error: int = ProjectSettings.save_custom(modloaderutils.get_override_path())
+
+		if is_setup_create_override_cfg:
```
Comment (Qubus0) [inline addons/mod_loader/mod_loader_setup.gd:63]: why is this here and not only in setup_modloader? shouldn't everything be set up already?

File: addons/mod_loader/mod_loader_setup.gd [hunk 2]
```diff
@@ -82,16 +87,18 @@ func setup_modloader() -> void:
...
+		handle_override_cfg()
+	else:
+		handle_project_binary()
+
 	modloaderutils.log_info("ModLoader setup complete", LOG_NAME)
 
-	create_project_binary()
-	inject_project_binary()
-	clean_up_project_binary_file()
+	modloaderutils.log_info("ModLoader setup complete", LOG_NAME)
```
Comment (Qubus0) [inline addons/mod_loader/mod_loader_setup.gd:101]: double log

File: addons/mod_loader/mod_loader_setup.gd (hunk 1 above)
Comment (KANAjetzt) [reply addons/mod_loader/mod_loader_setup.gd:63]: `ProjectSettings.set_setting(settings.IS_LOADER_SETUP_APPLIED, true)`

Only to save this setting. 
This will be fixed in one of the next PRs by using the method of https://github.com/GodotModding/godot-mod-loader/pull/96

File: addons/mod_loader/mod_loader_setup.gd (hunk 2 above)
Comment (otDan) [reply addons/mod_loader/mod_loader_setup.gd:101]: @KANAjetzt let me know when you changed this so I can approve again

Comment (KANAjetzt) [reply addons/mod_loader/mod_loader_setup.gd]: fixed 👍 @otDan
